lib/webtask.js

```diff
@@ -297,6 +303,15 @@ exports.issue_token = function (options /* req, claims, kid, keyset, store_code_
                     return cb(new Error('Error response from store_code_url when storing token. HTTP ' + res.statusCode + ': ' + (body || '<unknown>')));
                 cb();
             });
+        },
+        function (cb) {
+            if(new_claims.auth && !new_claims.jtn)
+                return cb(new Error('Can\'t set user authorization for unnamed webtasks.'));
```

Comment:
What do you think of calling these 'anonymous' webtasks? Just wondering your thoughts on the terminology.